parts/k8s/kubernetesagentresourcesvmas.t

```diff
@@ -12,7 +12,7 @@
 {{end}}
       ],
       "location": "[variables('location')]",
-      "name": "[concat(variables('{{.Name}}VMNamePrefix'), 'nic-', copyIndex(variables('{{.Name}}Offset')))]",
+      "name": "[concat(variables('{{.Name}}VMNamePrefix'), copyIndex(variables('{{.Name}}Offset')), '-nic1')]",
```

Comment:
Why the numbering here? Is this addressing a use case of several NICs?

Reply:
Not really, I thought if one day a one present itself you will already have a numbered nic.

Reply:
I think for now it's fine to leave it without numbering since there is only one per VM. If you do see value in numbering, it should start from 0 to be consistent with numbering of other resources.